reps/webpack.config.js

```diff
@@ -38,7 +39,7 @@ module.exports = {
   },
   resolve: {
     alias: {
-      "devtools": path.join(__dirname, "../../../../mozilla.org/fx-team/devtools"),
+      "devtools": path.join(__dirname, config.rootPath + '/devtools'),
```

Comment:
This should be:

`"devtools": path.join(__dirname, config.rootPath + '/fx-team/devtools'),`

Honza